doc/source/whatsnew/v0.22.0.txt

```diff
@@ -45,6 +45,7 @@ Other API Changes
 - :class:`Timestamp` will no longer silently ignore unused or invalid ``tz`` or ``tzinfo`` keyword arguments (:issue:`17690`)
 - :class:`Timestamp` will no longer silently ignore invalid ``freq`` arguments (:issue:`5168`)
 - :class:`CacheableOffset` and :class:`WeekDay` are no longer available in the ``pandas.tseries.offsets`` module (:issue:`17830`)
+- :function:`tseries.frequencies.get_freq_group` and :list:`tseries.frequencies.DAYS` are removed from the public API (:issue:`18034`)
```

Comment:
these tags are not going to work, nor are these in the api.rst so they won't reference anything, just use ``tseries.frequencies.get_freq_group()`` etc

Reply:
OK.  Does this take precedence over "don't push again"?

Reply:
no actually push again with this fix
everything else ok